package/MDAnalysis/analysis/align.py

```diff
@@ -630,7 +630,8 @@ def rms_fit_trj(
         tol_mass=0.1,
         strict=False,
         force=True,
-        quiet=False,
+        verbose=None,
+        quiet=None,
```

Comment:
While I see the point of not having quiet in the signature, I would prefer not using kwargs when not needed. Indeed, by adding kwargs to the signature makes us loose tests on the arguments that need to be reintroduced in the body of the function (i.e. catching invalid arguments). This adds quite a lot of complexity.

I can remove quiet from the signature, but I'd rather not.